api/reg.py

```diff
@@ -524,7 +525,11 @@ def beginFile(self, genOpts):
         #
         # User-supplied prefix text, if any (list of strings)
         if (genOpts.prefixText):
+            git_rev = subprocess.check_output(['git', 'rev-parse', '--short=10', 'HEAD']).decode('utf-8').strip()
```

Comment:
If `git` is not available, the python script fails, and the file isn't generated. If `git` fails, the script goes ahead and uses whatever output git gave (probably nothing, with this command) to perform the substitution.
I can change this if you want, just tell me what behaviour you'd like :)